javatools/__init__.py

```diff
@@ -1888,7 +1890,7 @@ def __eq__(self, other):
             return False
 
         # for each of the key/val pairs, check equality
-        for key, lval in self.items():
+        for key, lval in items(self):
```

Comment:
Whoops. That's true, this change is not needed.
I have corrected its commit, amended, and (again) rebased the `py3` branch on top of it.

I have not yet rebased `py3` on the recent `master` update, as the latter fails tests (because of minor mistype).